### ISSUE TITLE
adeptpower improve ability false maximum powerrating

### ISSUE THREAD

**Issue by chummer5a** [chummer5a/chummer5a#111]
<a href="https://github.com/GoogleCodeExporter"><img src="https://avatars.githubusercontent.com/u/9614759?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [GoogleCodeExporter](https://github.com/GoogleCodeExporter)**
_Tuesday Mar 17, 2015 at 08:56 GMT_
_Originally opened as https://github.com/chummer5a/chummer5/issues/37_

----

```
What steps will reproduce the problem?
1. create mystic adept
2. set blades skill to 6
3. add improve ability(blades) adept power

What is the expected output? What do you see instead?
i ecepted that the adept power would max out at 3 (max for this power is final 
skill level 1,5x of base skill level which would be 9; skill blades at 6 + 
power at 3 makes 9. Instead i can increase the power to 6 maxing out at magic 
attribute.

What version of the product are you using? On what operating system?
0.0.5.139 // Windows 7

Please provide any additional information below.
adept power critical strike has no powerrating and is stuck at level 1

```

Original issue reported on code.google.com by `Ti...@gmx.de` on 16 Jan 2015 at 1:34

**Comment by chummer5a** [comment]
Improved Ability has no validation against the skill level it's increasing.

Critical Strike behaves differently in SR5 to SR4, and may only be selected multiple times for different weapons. Working as intended.

**Comment by chummer5a** [comment]
Partially resolved in 5.159. Need to filter the available skills to skills that the character possesses, but the quality partially breaks if you try to load it anyway, so it's something of a moot point. Review at a later milestone.
